test/unit/modules/uploader.js

```diff
@@ -0,0 +1,43 @@
+import Uploader from '../../../modules/uploader';
+import { Range } from '../../../core/selection';
+
+describe('Uploader', function() {
+  describe('image uploading', function() {
+    it('Check file types upload filtration', function() {
+      const testRange = new Range(0);
+      let uploads;
+
+      const quillMock = {
+        root: {
+          addEventListener: () => {},
+        },
+      };
+
+      const gifFile = {
+        name: 'test.gif',
+        type: 'image/gif',
+      };
+
+      const pngFile = {
+        name: 'test.png',
+        type: 'image/png',
+      };
```

Comment:
```
      const gifFile = {
        name: 'test.gif',
        type: 'image/gif',
      };

      const pngFile = {
        name: 'test.png',
        type: 'image/png',
      };
```
You can run simple test for each image type. It would be nice, if you check not supported type too

```
['gif', 'png'].forEach((image) => {
  it(`upload ${image}`, function() {
    ...
  })
}); 
```